plugins/callback/log_plays.py

```diff
@@ -79,30 +80,33 @@ def log(self, host, category, data):
                 if invocation is not None:
                     data = json.dumps(invocation) + " => %s " % data
 
-        path = os.path.join(self.log_folder, host)
+        path = os.path.join(self.log_folder, result._host.name)
```

Comment:
```suggestion
        path = os.path.join(self.log_folder, result._host.get_name())
```
This is what the default plugin does.

Reply:
That was kind of confusing, as half of the callbacks use `_host.name`, and the other half `_host.get_name()`.
Fixed.